components/properties/src/maps.rs

```diff
@@ -61,6 +61,13 @@ where
     get_cp_map(provider, key::GENERAL_CATEGORY_V1)
 }
 
+pub fn get_bidi_class<D>(provider: &D) -> CodePointMapResult<BidiClass>
```

Comment:
You need to do 2 things here:

1. After changing `BidiClass` from a Rust enum to a newtype (see below), then you need to import `BidiClass` at the top of the file.  `cargo check --tests` is the quickest way to get compiler errors & suggestions for things like this.
2. You also need to implement the `TrieValue` trait for `BidiClass`, that goes into [`trievalue.rs`](https://github.com/unicode-org/icu4x/blob/main/components/properties/src/trievalue.rs).  Use `WordBreak` as an example of what that looks like.

Reply:
done, thanks :)